motion_control/motion_control_common/ThrottledController.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2025 COGIP Robotics association <cogip35@gmail.com>
+// This file is subject to the terms and conditions of the GNU Lesser
+// General Public License v2.1. See the file LICENSE in the top level
+// directory for more details.
+
+#include "motion_control_common/ThrottledController.hpp"
+
+#define ENABLE_DEBUG 0
+#include <debug.h>
+#include <inttypes.h>
+
+namespace cogip {
+
+namespace motion_control {
+
+ThrottledController::ThrottledController(BaseController* wrapped_controller,
+                                         uint32_t period_divider)
```

Comment:
Maybe you can use a smaller type. `uint16_t` is enough I think for all kind of use cases. But this is a matter of optimisation, so feel free to ignore this comment. (Maybe a `uint8_t` can also do the job)